packages/styles/src/link/index.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  align-items: center;
+  color: var(--syn-link-color);
+  display: inline-flex;
+  font: var(--syn-body-medium-regular);
```

Comment:
Per default we should neither have a size or a weight here. If someone drops a syn-link inside a text, it should just apply the blue color and underline, but not change any size. And if someone wants to make it bold, it should just work.

Reply:
Done by using font-size / font-family